### PR TITLE
Custom Footer

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
**all changes made under the following filepaths do not need to be reviewed as they are just document migration:**
- `docs/community`
- `docs/community/deprecated/tutorials-contest`

staged: https://meag-footer.avalanche-docs.pages.dev/

## Why this should be merged

Giving our docs site a custom footer provides the user with a better experience and frees up the main components (such as navbar and sidebar) for the content we are trying to deliver: technical documentation and tutorials.

## How this works

[Swizzled ](https://docusaurus.io/docs/swizzling) the standard Footer that comes as part of Docusaurus' framework and rebuilt using React and react-icons to provide more social links, support, links to Core and the Subnet Explorer, and various miscellaneous links.

Before:
image

After: 
image

The 'Community' navbar option was removed and the tutorials contest preserved via link in the footer. Eventually these docs will be fully deprecated. 

## How these changes were tested 

`yarn start`
`yarn build`
all links tested

## Workflow checks

- [x] ran `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [x] ran `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [x] completed the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass
- [x] if a doc was removed/deleted, the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [x] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -222,4 +199,105 @@ li hr {
...
+
+.footer-heading {
+  font-weight: "800";
+  font-size: large;
+  margin-bottom: 2px;
+  color: var(--ifm-color-primary)
+}
+
+.footer-list {
+  font-weight: "600";
```
Comment (nahoc10) [inline src/css/custom.css:261]: usually font-weight is not a string like this, should be: `font-weight: 600;`

File: src/css/custom.css [hunk 2]
```diff
@@ -222,4 +199,105 @@ li hr {
...
   padding-left: 7px;
+}
+
+/* Custom Footer */
+.theme-doc-sidebar-menu{
+  font-size: 13px;
+}
+
+.footer {
+  display: 'flex';
```
Comment (nahoc10) [inline src/css/custom.css:210]: shouldn't be string, just `flex;`

File: src/css/custom.css [hunk 3]
```diff
@@ -222,4 +199,105 @@ li hr {
...
+  justify-content: 'center';
+  align-items: 'center';
```
Comment (nahoc10) [inline src/css/custom.css:212]: shouldn't be strings, just `flex;`, etc

File: src/css/custom.css [hunk 4]
```diff
@@ -222,4 +199,105 @@ li hr {
...
+  width: 28px;
+}
+
+.footer-social:hover {
+  color: var(--ifm-color-primary);
+  transition: color 0.15s, color 0.15s;
+}
+
+.footer-heading {
+  font-weight: "800";
```
Comment (nahoc10) [inline src/css/custom.css:254]: same here

File: src/pages/index.js [hunk 5]
```diff
@@ -10,9 +10,16 @@ function Home() {
...
-      <main>
+      <main className={styles.main}>
         <br />
-        <h1 align="center" style={{ fontWeight: "750" }}>
+        <h1
+          align="center"
+          style={{
+            fontWeight: "700",
+            marginBottom: "0px",
+            fontSize: "x-large",
```
Comment (nahoc10) [inline src/pages/index.js:20]: what's `x-large`?

Comment (meaghanfitzgerald) [reply src/pages/index.js:20]: honestly not sure, was a suggestion from my css plugin

**Comment by yulin-dong** [comment]
My 2 cents: the bottom `Avalanche` makes the page crowded. I would suggest to remove it
Screenshot 2023-08-11 at 7 51 27 AM

File: src/pages/index.js (hunk 5 above)
Comment (meaghanfitzgerald) [reply src/pages/index.js:20]: Docusaurus default styling if inherited from the [Infima](https://infima.dev/) styling framework so values might be from there?

**Pull request merged**